src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -171,34 +189,42 @@ private void handleExit() {
      * Changes theme to light theme
      */
```

Comment:
Don't forget to update the Javadoc. Perhaps `toggles between light and dark theme` fits this better.

Reply:
Oh yea, thanks for the catch.